packages/synapse-react-client/src/components/widgets/RadioGroup.tsx

```diff
@@ -17,34 +17,34 @@ export function RadioGroup<T extends string | boolean | number = string>(
     : `radiogroup`
 
   return (
-    <div className={className} role="radiogroup">
+    <div id={props.id} className={className} role="radiogroup">
```

Comment:
Should an `id` always be specified in the `div`? If so, should an id be generated with `useId` when `props.id` is undefined?

Reply:
I put it in the div since we weren't using the `id` prop otherwise. I don't think it would cause any problems to not have an ID here--I'd probably try to get rid of it entirely instead

Reply:
I decided to completely remove the prop